packages/headless/src/test/index.ts

```diff
@@ -1,4 +1,17 @@
-export * from './mock-engine';
-export * from './mock-state';
-export * from './mock-result';
-export * from './mock-raw';
+export {
```

Comment:
That's something we should have caught on earlier I guess.

It seems we use that for Atomic, so we are guilty of the same thing we should not encourage clients to do... 

That is, treating internal stuff from a package (Headless) as something that can be seen as "public" and backward compatible.

Reply:
Glad not too much stuff was exposed in the end 😅